docs/scenes/get-started/set-up-your-environment/instructions/_androidStudioEnvironmentInstructions.mdx

```diff
@@ -0,0 +1,147 @@
+import { Step } from '~/ui/components/Step';
+import { Terminal } from '~/ui/components/Snippet';
+import { Tabs, Tab } from '~/ui/components/Tabs';
+
+## Install Watchman and JDK
+
+<Tabs>
+
+<Tab label='macOS'>
+
+To set up your development environment on macOS for Android, you will install Watchman, JDK, and Android Studio.
+
+#### Prerequisites
+
+Use a package manager such as [Homebrew](https://brew.sh/) to install the following dependencies.
+
+#### Install dependencies
+
+<Step label="1">
+
+[Install Watchman](https://facebook.github.io/watchman/docs/install#macos) using a tool such as Homebrew:
+
+<Terminal cmd={['$ brew install watchman']} />
+
+</Step>
+
+<Step label="2">
+
+Install OpenJDK distribution called Azul Zulu using Homebrew. This distribution offers JDKs for both Apple Silicon and Intel Macs.
+
+<Tabs>
+
+<Tab label="SDK 50 and above">
+
+We recommend JDK 17. Run the following commands in a terminal window to install it:
+
+<Terminal
+  cmd={['$ brew tap homebrew/cask-versions', '$ brew install --cask zulu@17']}
+  cmdCopy="brew tap homebrew/cask-versions && brew install --cask zulu@17"
+/>
+
+After you install the JDK, add the `JAVA_HOME` environment variable in **~/.bash_profile** (or **~/.zshrc** if you use Zsh):
+
+```bash
+export JAVA_HOME=/Library/Java/JavaVirtualMachines/zulu-17.jdk/Contents/Home
+```
+
+</Tab>
+
+<Tab label="SDK 49 and below">
```

Comment:
We can remove SDK 49 and below instructions since this whole section to the current SDK. After that, we'll also don't have to use `Tabs` here. Can just include Java17 instructions.

Similar changes can be done for Windows down below.